app/scripts/views/settings/client_disconnect.js

```diff
@@ -0,0 +1,128 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const Cocktail = require('cocktail');
+  const FloatingPlaceholderMixin = require('views/mixins/floating-placeholder-mixin');
+  const FormView = require('views/form');
+  const ModalSettingsPanelMixin = require('views/mixins/modal-settings-panel-mixin');
+  const SignedOutNotificationMixin = require('views/mixins/signed-out-notification-mixin');
+  const t = require('views/base').t;
+  const Template = require('stache!templates/settings/client_disconnect');
+
+  const REASON_SELECTOR = '.disconnect-reasons';
+  const REASON_HELP = {
+    'lost': t('We\'re sorry to hear about this. You should change your Firefox Account password, and look for ' +
+      'information from your device manufacturer about erasing your data remotely.'),
+    'suspicious': t('We\'re sorry to hear about this. If this was a device you really don\'t trust, you should change your ' +
+      'Firefox Account password, and change any passwords saved in Firefox.')
+  };
+
+  var View = FormView.extend({
+    template: Template,
+    className: 'clients-disconnect',
+    viewName: 'settings.clients.disconnect',
+
+    events: {
+      'change select': 'selectOption',
+      'click': 'closePanelIfDisconnected'
+    },
+
+    initialize () {
+      // user is presented with an option to disconnect device
+      this.toDisconnect = true;
+    },
+
+    beforeRender () {
+      // receive the device collection and the item to delete
+      // if deleted the collection will be automatically updated in the settings panel.
+      let clients = this.model.get('clients');
+      let deviceId = this.model.get('itemId');
```

Comment:
Niether `clients` nor `itemId` are used past this point, would it be simpler to just pass in the expected client from `clients.js`?